lib/model/picture.dart

```diff
@@ -1,20 +1,22 @@
-class Picture{
-
+class Picture {
```

Comment:
overall is good, but more modern approach in flutter which is used almost everywhere - freezed library: https://pub.dev/packages/freezed . Which allow you to have all these from/toMap methods out from the box and all of this stuff will be done by code generation